flake8 update 3.8.1 lint failures

```
modin/pandas/__init__.py:22:9: F523 '...'.format(...) has unused arguments at position(s): 0
modin/pandas/test/test_dataframe.py:2453:17: E741 ambiguous variable name 'l'
modin/pandas/test/test_series.py:1874:13: E741 ambiguous variable name 'l'
modin/engines/base/frame/data.py:1138:39: E741 ambiguous variable name 'l'
modin/engines/base/frame/data.py:1151:39: E741 ambiguous variable name 'l'
```

--- a/modin/engines/base/frame/data.py
+++ b/modin/engines/base/frame/data.py
@@ -1135,7 +1135,7 @@ class BasePandasFrame(object):
             left_parts = self._partitions
             right_parts = [o._partitions for o in others]
             new_lengths = self._row_lengths + [
-                l for o in others for l in o._row_lengths
+                length for o in others for length in o._row_lengths
             ]
             new_widths = self._column_widths
         elif (
@@ -1148,7 +1148,7 @@ class BasePandasFrame(object):
             right_parts = [o._partitions for o in others]
             new_lengths = self._row_lengths
             new_widths = self._column_widths + [
-                l for o in others for l in o._column_widths
+                length for o in others for length in o._column_widths
             ]
         else:
             left_parts, right_parts, joined_index = self._copartition(

--- a/modin/pandas/__init__.py
+++ b/modin/pandas/__init__.py
@@ -19,8 +19,10 @@ if pandas.__version__ != __pandas_version__:
     import warnings
 
     warnings.warn(
-        "The pandas version installed does not match the required pandas version in "
-        "Modin. This may cause undesired side effects!".format(__pandas_version__)
+        "The pandas version installed {} does not match the supported pandas version in"
+        " Modin {}. This may cause undesired side effects!".format(
+            pandas.__version__, __pandas_version__
+        )
     )
 
 from pandas import (

--- a/modin/pandas/test/test_dataframe.py
+++ b/modin/pandas/test/test_dataframe.py
@@ -2450,19 +2450,19 @@ class TestDataFrameDefault:
         if name_contains(request.node.name, numeric_dfs):
             # We have to test this way because equality in plots means same object.
             zipped_plot_lines = zip(modin_df.plot().lines, pandas_df.plot().lines)
-            for l, r in zipped_plot_lines:
-                if isinstance(l.get_xdata(), np.ma.core.MaskedArray) and isinstance(
-                    r.get_xdata(), np.ma.core.MaskedArray
+            for left, right in zipped_plot_lines:
+                if isinstance(left.get_xdata(), np.ma.core.MaskedArray) and isinstance(
+                    right.get_xdata(), np.ma.core.MaskedArray
                 ):
-                    assert all((l.get_xdata() == r.get_xdata()).data)
+                    assert all((left.get_xdata() == right.get_xdata()).data)
                 else:
-                    assert np.array_equal(l.get_xdata(), r.get_xdata())
-                if isinstance(l.get_ydata(), np.ma.core.MaskedArray) and isinstance(
-                    r.get_ydata(), np.ma.core.MaskedArray
+                    assert np.array_equal(left.get_xdata(), right.get_xdata())
+                if isinstance(left.get_ydata(), np.ma.core.MaskedArray) and isinstance(
+                    right.get_ydata(), np.ma.core.MaskedArray
                 ):
-                    assert all((l.get_ydata() == r.get_ydata()).data)
+                    assert all((left.get_ydata() == right.get_ydata()).data)
                 else:
-                    assert np.array_equal(l.get_xdata(), r.get_xdata())
+                    assert np.array_equal(left.get_xdata(), right.get_xdata())
 
     def test_replace(self):
         data = test_data_values[0]

--- a/modin/pandas/test/test_series.py
+++ b/modin/pandas/test/test_series.py
@@ -1871,19 +1871,19 @@ def test_plot(request, data):
     if name_contains(request.node.name, numeric_dfs):
         # We have to test this way because equality in plots means same object.
         zipped_plot_lines = zip(modin_series.plot().lines, pandas_series.plot().lines)
-        for l, r in zipped_plot_lines:
-            if isinstance(l.get_xdata(), np.ma.core.MaskedArray) and isinstance(
-                r.get_xdata(), np.ma.core.MaskedArray
+        for left, right in zipped_plot_lines:
+            if isinstance(left.get_xdata(), np.ma.core.MaskedArray) and isinstance(
+                right.get_xdata(), np.ma.core.MaskedArray
             ):
-                assert all((l.get_xdata() == r.get_xdata()).data)
+                assert all((left.get_xdata() == right.get_xdata()).data)
             else:
-                assert np.array_equal(l.get_xdata(), r.get_xdata())
-            if isinstance(l.get_ydata(), np.ma.core.MaskedArray) and isinstance(
-                r.get_ydata(), np.ma.core.MaskedArray
+                assert np.array_equal(left.get_xdata(), right.get_xdata())
+            if isinstance(left.get_ydata(), np.ma.core.MaskedArray) and isinstance(
+                right.get_ydata(), np.ma.core.MaskedArray
             ):
-                assert all((l.get_ydata() == r.get_ydata()).data)
+                assert all((left.get_ydata() == right.get_ydata()).data)
             else:
-                assert np.array_equal(l.get_xdata(), r.get_xdata())
+                assert np.array_equal(left.get_xdata(), right.get_xdata())
 
 
 @pytest.mark.parametrize("data", test_data_values, ids=test_data_keys)

